ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -108,28 +111,57 @@ Set<String> getRecentNotFoundContainersCache() {
   synchronized void fetchAndUpdateCache() throws SQLException {
     cachedAccountService.fetchAndUpdateCache();
     if (config.enableNewDatabaseForMigration) {
-      cachedAccountServiceNew.fetchAndUpdateCache();
+      try {
+        cachedAccountServiceNew.fetchAndUpdateCache();
+      } catch (SQLException e) {
+        if (config.ignoreNewDatabaseUploadError) {
```

Comment:
I am curious why we felt the need for this. Do we observe failures on the new DB? If so, it should be a concern when it becomes SOT

Reply:
This can be used at early stage where we don't want the new db upload affect the old db upload until we verify everything is good.

Reply:
Would you disable `config.ignoreNewDatabaseUploadError` before setting `config.enableGetFromNewDbOnly` to `true` since we would need to catch upload errors at that time?

Reply:
yes, I'll disable config.ignoreNewDatabaseUploadError before setting config.enableGetFromNewDbOnly. so the step would be:

support dual upload
one time dump data
once it's in sync and don't see any issue, disable config.ignoreNewDatabaseUploadError
after a while when it's stable, enable config.enableGetFromNewDbOnly to get from new db only
disable upload to old db.